scripts/run-system-test.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+DEBUG_LOG_PATH=./debug_log.txt
+# upgrade gradle to 5.0
+wget http://services.gradle.org/distributions/gradle-5.0-all.zip > /dev/null
+unzip gradle-5.0-all.zip > ${DEBUG_LOG_PATH}
+./gradle-5.0/bin/gradle -v
```

Comment:
should install gradle in traivis.yml using before_script, then we can run system test in local machine

Reply:
fixed